### DISCUSSION TITLE
[Feature Req] Automatic Silencing for Draft PRs

### DISCUSSION THREAD

**Discussion by shaneutt** [community/community#27310] [Other Feature Feedback, Questions, & Ideas]
The purpose of this issue to to request the ability to silence draft PR notifications _without_ unsubscribing from them: the effect should be while something is in draft I get no updates about it but the moment it gets out of draft the updates start coming in. This is particularly valuable for reducing noise for someone who gets several hundred notifications every week and doesn't want to review draft PRs until they're marked `ready`.

Original Post:

My day to day workflow currently as a tech lead revolves around Github Notifications. I have not been able to find a setting or option that would **disable notifications temporarily for PRs which are in a draft state, and then notify me and re-enabling notifications when they reach a ready state**, thus reducing notification noise because draft PRs tend to get a lot of quick updates as the writer is often assuming nobody is reviewing yet (sometimes they're even just exercising CI first). Does such a feature exist?

**Comment by sawyerbivens** [comment]
Not sure if this is the answer you're looking for, but the only notification settings can be found [here](https://github.com/settings/notifications). There doesn't seem to be anything for not getting notifications for draft state PRs and vise verse. I think Github should add something like though because notifications get filled extremely quickly in my experience.

Anyways in short, no such a feature does not exist.

**Reply by shaneutt** [reply]
Thanks for your answer. Where is the best place to request such a feature?

**Reply by sawyerbivens** [reply]
Honestly I think the best place to request a feature in general is here on the community discussions, just start a discussion with the title prefixed with "[Feature Req]" or something like that and then hope someone at Github sees it and can pass it along. As far as I'm aware there's not anything like a form to just submit a feature request.

**Reply by shaneutt** [reply]
Sounds good, I re-used this discussion.

**Comment by shaneutt** [comment]
Turns out if the API would simply expose the ability to mark a thread as "done", I could implement a lot of what I'm looking for myself. Relevant to https://github.com/community/community/discussions/15669
